configuration/configuration.go

```diff
@@ -86,6 +86,12 @@ type Configuration struct {
 			ClientCAs []string `yaml:"clientcas,omitempty"`
 		} `yaml:"tls,omitempty"`
 
+		// Headers is a set of headers to include in HTTP responses. A common
+		// use case for this would be security headers such as
+		// Strict-Transport-Security. The map keys are the header names, and
+		// the values are the associated header payloads.
+		Headers map[string]string `yaml:"headers,omitempty"`
```

Comment:
You're right - that was silly on my part.

The notifications item is `[]Endpoint`, so to match that, we would want `[]map[string]string`, right? And if I understand yaml syntax correctly, it would look like this:

    headers:
        - X-Header-1: value-a
        - X-Header-1: value-b
        - X-Header-2: value-x

The alternative of using `http.Header` would mean syntax like:

    headers:
        X-Header-1:
            - value-a
            - value-b
        X-Header-2:
            - value-x

...which looks a little weird to me.